.github/workflows/ci.yml

```diff
@@ -101,22 +101,26 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-${{ matrix.compiler }}-${{matrix.build_type}}-${{matrix.generator}}-${{matrix.developer_mode}}
 
+      - uses: actions/setup-python@v2
+
+      - run: pip3 install cmake ninja conan gcovr
```

Comment:
On the GitHub images, it happens that `pip3` is on the path. But a lot of the time, it is actually called `pip`, which means it is needed to check if `pip` is version 3 or above. 

https://github.com/aminya/setup-cpp/blob/94827f017e0e4445837e4cc099daf7ce849a059c/src/utils/setup/setupPipPack.ts#L19-L29

Reply:
> On the GitHub images, it happens that `pip3` is on the path. But a lot of the time, it is actually called `pip`, which means it is needed to check if `pip` has version 3 or above.
> 

But isn't this yml file *only* used on github images?

It's specifically in `.github/workflows/ci.yml`, so shouldn't that be the primary concern here?